cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1149,538 @@ def cancel(self, req):
+                r, _, e = bash.bash_roe(cmd_str)
+                if r != 0:
+                    raise Exception("mkdir failed: %s" % e)
+
+            def copy_file():
+                cmd_str = 'sshpass -f %s scp -P %d -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null %s %s@%s:%s' % (
+                    quoted_ssh_passwd_file, target_host_ssh_port,
+                    upgrade_package_source_path, quoted_ssh_username,
+                    quoted_ssh_target_ip, linux.shellquote(target_upgrade_package_path))
+                r, _, e = bash.bash_roe(cmd_str)
+                if r != 0:
+                    raise Exception("scp failed: %s" % e)
```

Ruff (0.15.4)

[warning] Abstract `raise` to an inner function

(TRY301)